cmd/tink-server/Dockerfile

```diff
@@ -1,10 +1,16 @@
+FROM golang:1.15-alpine
+RUN apk add --no-cache --update --upgrade ca-certificates make git
+COPY . /usr/myapp
+WORKDIR /usr/myapp
+RUN make server
+
 FROM alpine:3.11
+COPY --from=0 /usr/myapp/cmd/tink-server/tink-server /usr/bin/tink-server
 
-ENTRYPOINT ["tink-server"]
 EXPOSE 42113
 EXPOSE 42114
 
 RUN apk add --update ca-certificates && \
     apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing cfssl
 
-COPY tink-server /bin/
+ENTRYPOINT ["/usr/bin/tink-server"]
```

Comment:
I think lets keep the image "metadata" together at the top and then the contents after so it would be like:

```
FROM alpine:3.11

EXPOSE 42113
EXPOSE 42114
ENTRYPOINT ["/usr/bin/tink-server"]

RUN apk add ...
COPY --from=0 ...
```

Notice I re-arranged the COPY, RUN -> RUN, COPY so we can re-use the RUN apk cached entry.